home-api-tests/src/test/java/com/softserveinc/ita/homeproject/api/tests/utils/MailHogUtil/Dto/FromDto.java

```diff
@@ -0,0 +1,16 @@
+package com.softserveinc.ita.homeproject.api.tests.utils.MailHogUtil.Dto;
```

Comment:
Please pay attention to the package naming. You violated defined conventions.

Reply:
Can we also go away of affiliation on some specific email mock implementation and at least have general naming? Moreover, I suggest you try to utilize a facade pattern here https://refactoring.guru/design-patterns/facade So main goal is to have the ability to quickly switch to another email mock server in terms of our tests should be loosely coupled with any specific mocks.

Reply:
Fixed conventions, added facade pattern, not sure if this is what you wanted to see, please review again

Reply:
More or less =)